app/src/org/commcare/views/ViewUtil.java

```diff
@@ -195,4 +196,17 @@ private static void divideIndeterminateSpace(int[] widths, int fullSize,
             }
         }
     }
+
+
+    /**
+     * Adds RTL support to the textview. It will make the textView ALIGN_LEFT if the view’s resolved
+     * layoutDirection is LTR, and ALIGN_RIGHT otherwise.
+     */
+    public static void addRTLSupport(TextView textView) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
```

Comment:
We can also probably have add these to the `CommonTheme` which will make all the textviews RTL supportive. 
But then I guess, we'll have to find out all the buttons/textviews which could be broken because of `view_start` textDirection, and will have to manually change the text direction for them.